### PR TITLE
Add docstrings to unit tests in recover_from_sig

### PR DESCRIPTION
**Pull request by pirapira**
This is a small beginning of https://github.com/raiden-network/raiden-contracts/issues/357 .

### CODE REVIEW

File: raiden_contracts/tests/unit/test_recover_from_signature.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ def test_verify(
         create_channel,
         create_balance_proof,
 ):
+    """ the signature test contract returns the correct addresses on the balance hash signed
+    by both ends of a channel """
```
Comment (loredanacirstea) [inline raiden_contracts/tests/unit/test_recover_from_signature.py]: Please follow the docstring rules from https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md.
E.g.:
```
    """ Check if ECVerify.ecverify returns the correct address

    This checks whether ECVerify.ecverify returns the correct addresses on the balance hash signed
    by both ends of a channel.
    """
```

File: raiden_contracts/tests/unit/test_recover_from_signature.py [hunk 2]
```diff
@@ -71,6 +75,7 @@ def test_ecrecover_output(
         get_accounts, create_channel,
         create_balance_proof,
 ):
+    """ ecrecover returns the address who signed a balance proof """
```
Comment (loredanacirstea) [inline raiden_contracts/tests/unit/test_recover_from_signature.py]: `who signed` -> `that was used to sign`

File: raiden_contracts/tests/unit/test_recover_from_signature.py (hunk 1 above)
Comment (pirapira) [reply raiden_contracts/tests/unit/test_recover_from_signature.py]: Done this here and there!

File: raiden_contracts/tests/unit/test_recover_from_signature.py (hunk 2 above)
Comment (pirapira) [reply raiden_contracts/tests/unit/test_recover_from_signature.py]: Applied this change.

**Comment by pirapira** [comment]
@loredanacirstea ready for merge?

**Pull request merged**